docs/visual-basic/reference/command-line-compiler/compiler-options-listed-alphabetically.md

```diff
@@ -76,6 +76,5 @@ The Visual Basic command-line compiler is provided as an alternative to compilin
   
 ## See also
 - [Visual Basic Compiler Options Listed by Category](../../../visual-basic/reference/command-line-compiler/compiler-options-listed-by-category.md)
-- [Introduction to the Project Designer](https://msdn.microsoft.com/library/898dd854-c98d-430c-ba1b-a913ce3c73d7)
```

Comment:
Perhaps link to this one instead?
https://docs.microsoft.com/en-us/visualstudio/ide/managing-project-and-solution-properties?view=vs-2017

Also can you remove the following two C# links from here too?